examples.js

```diff
@@ -166,8 +166,8 @@ function update(id) {
  * Loads a language, including all dependencies
  *
  * @param {string} lang the language to load
- * @type {Promise} the promise which resolves as soon as everything is loaded
- */
+ * @returns {Promise<Event|void>} the promise which resolves as soon as everything is loaded
```

Comment:
Missing `.` after `Promise`?

Reply:
It’s unnecessary, also I use the [TypeScript style syntax for JSDoc types](https://github.com/Microsoft/TypeScript/wiki/JsDoc-support-in-JavaScript).

Reply:
Seems like JSDoc is also able to handle this syntax.